### PR TITLE
Upgrade Orchard core packages

### PR DESCRIPTION
**Pull request by MylesB93**

### CODE REVIEW

File: Etch.OrchardCore.CivicCookieControl.csproj [hunk 1]
```diff
@@ -21,17 +21,17 @@
...
-    <PackageReference Include="OrchardCore.ContentFields" Version="1.7.2" />
-    <PackageReference Include="OrchardCore.ContentManagement" Version="1.7.2" />
-    <PackageReference Include="OrchardCore.ContentManagement.Display" Version="1.7.2" />
-    <PackageReference Include="OrchardCore.DisplayManagement" Version="1.7.2" />
-    <PackageReference Include="OrchardCore.Flows" Version="1.7.2" />
-    <PackageReference Include="OrchardCore.Navigation" Version="1.7.2" />
-    <PackageReference Include="OrchardCore.ResourceManagement" Version="1.7.2" />
-    <PackageReference Include="OrchardCore.Module.Targets" Version="1.7.2" />
-    <PackageReference Include="OrchardCore.Title" Version="1.7.2" />
+    <PackageReference Include="Etch.OrchardCore.Fields" Version="1.7.0" />
```
Comment (MylesB93) [inline Etch.OrchardCore.CivicCookieControl.csproj:24]: @peterkeating Should I have upgraded the fields package?

Comment (peterkeating) [reply Etch.OrchardCore.CivicCookieControl.csproj:24]: Yes! :)

**Review by peterkeating — CHANGES_REQUESTED** [review]
Can you update the Orchard version specified in the README please.

**Review by peterkeating — APPROVED** [review]

File: Migrations.cs [hunk 2]
```diff
@@ -36,16 +36,16 @@ public Migrations(IContentDefinitionManager contentDefinitionManager)
...
+        public int CreateAsync()
         {
-            UpdateFrom1();
+            UpdateFrom1Async();
 
             return 2;
         }
 
-        public int UpdateFrom1()
+        public int UpdateFrom1Async()
```
Comment (peterkeating) [inline Migrations.cs]: This needs to change so it returns a `Task<int>` and is marked as async as shown below.

```
public async Task<int> UpdateFrom1Async() 
{
    ...
}
```

File: Migrations.cs [hunk 3]
```diff
@@ -36,16 +36,16 @@ public Migrations(IContentDefinitionManager contentDefinitionManager)
...
         {
-            _contentDefinitionManager.AlterPartDefinition(nameof(Models.CivicCookieControl), builder => builder
+            _contentDefinitionManager.AlterPartDefinitionAsync(nameof(Models.CivicCookieControl), builder => builder
```
Comment (peterkeating) [inline Migrations.cs]: Need an `await` before calls to an async method, as shown below.

```
await _contentDefinitionManager.AlterPartDefinitionAsync(nameof(Models.CivicCookieControl), builder => builder
```

File: Migrations.cs [hunk 4]
```diff
@@ -36,16 +36,16 @@ public Migrations(IContentDefinitionManager contentDefinitionManager)
 
         #region Migrations
 
-        public int Create()
+        public int CreateAsync()
```
Comment (peterkeating) [inline Migrations.cs]: Same as below, this should be `async Task<int>`.

File: Migrations.cs [hunk 5]
```diff
@@ -36,16 +36,16 @@ public Migrations(IContentDefinitionManager contentDefinitionManager)
...
         {
-            UpdateFrom1();
+            UpdateFrom1Async();
```
Comment (peterkeating) [inline Migrations.cs]: Missing `await` before call to method.

**Review by peterkeating — CHANGES_REQUESTED** [review]

**Pull request merged**